src/riak_core_apl.erl

```diff
@@ -76,38 +82,57 @@ get_apl_chbin(DocIdx, N, CHBin, UpNodes) ->
 %% for a given ring/upnodes list
 -spec get_apl(docidx(), n_val(), ring(), [node()]) -> preflist().
 get_apl(DocIdx, N, Ring, UpNodes) ->
-    [{Partition, Node} || {{Partition, Node}, _Type} <- 
+    [{Partition, Node} || {{Partition, Node}, _Type} <-
                               get_apl_ann(DocIdx, N, Ring, UpNodes)].
 
-%% Get the active preflist taking account of which nodes are up
-%% and annotate each node with type of primary/fallback
+%% Get the active preflist taking account of which nodes are up and
```

Comment:
@cmeiklejohn updated. realized i didn't have a @doc there b/c that file is completely lacking of them. Will one day go through and fix up I reckon.